mono/metadata/sgen-bridge.c

```diff
@@ -476,7 +476,7 @@ sgen_bridge_processing_finish (int generation)
 	if (compare_bridge_processors ())
 		compare_to_bridge_processor.processing_after_callback (generation);
 
-	mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_GC, "GC_BRIDGE: Complete, was running for %.2fms", mono_time_since_last_stw () / 10000.0f);
+	mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_GC, "GC_BRIDGE: Complete, was running for %.2fms", mono_time_since_last_stw () / 10000.0f);
```

Comment:
Not sure about this change. I think the GC stuff might be valuable.

Reply:
This one will show up on android, and it doesn't really pop out as a red flag if you're diagnosing some problem - I guess unless you suspect the gc bridge processing is running at the wrong time - not sure how often that's an issue for people.  I think this one is better at DEBUG.